thunderbird/processes/wps_generate_climos.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
                 "climo",
                 "Climatological Period",
                 abstract="Year ranges",
-                min_occurs=1,
+                min_occurs=0,
                 mode=0,
                 allowed_values=["all"]
```

Comment:
Since we are using `standard_climo_periods` we don't need to manually build these options.  Use `standard_climo_periods` here.